mule-deployer/src/main/java/org/mule/tools/model/anypoint/CloudHubDeployment.java

```diff
@@ -37,6 +37,9 @@ public class CloudHubDeployment extends AnypointDeployment {
   @Parameter
   protected Boolean persistentQueues = false;
 
+  @Parameter
+  protected Boolean customLog4J = false;
```

Comment:
I'm not sure how this works on cloudhub but it seems that this name can be confusing for the customers. Because in Cloudhub UI this same functionality is presented as "Disable Cloudhub logs". can we name this parameter more similar to the UI parameter?